### DISCUSSION TITLE
SNOMED code deprecation - mapping SOURCE to STANDARD

### DISCUSSION THREAD

**Discussion by lizzieLepoo** [OHDSI/Vocabulary-v5.0#836] [Q&A] [answered]
Hi - I've had a look at other posts re issues etc relating to deprecation and wasn't sure whether to start afresh as nothing seemed quite the same angle.

Where SNOMED CT replace a code (eg allergy to latex) - what is the best reliable route to use CONCEPT and CONCEPT_RELATIONSHIP to ensure that I get always and only the appropriate new standard concept IDs for the now-deprecated codes?

I've had a dig around and can see that using the relationships "Concept poss_eq to" or "Concept poss_eq from" might be useful

However, where there's a direct replacement for one or more deprecated codes to a new standard active SNOMED code - is there possibly a better relationship that could be created to show this more definite, direct linkage - and so aid in the SOURCE_TO_STANDARD mappings for deprecated codes?

Thanks

**Comment by cgreich** [comment, marked as answer]
@lizzieLepoo:

Want to have this discussion in the Forum, so other people can participate? In the Vocabulary category?

**Reply by lizzieLepoo** [reply]
Sure thing. Still very much learning what to post where! I will go put this there, Thank you.
